packages/ui/components/form/select/Select.tsx

```diff
@@ -1,13 +1,14 @@
 import { useId } from "@radix-ui/react-id";
 import * as React from "react";
-import type { GroupBase, Props, SingleValue, MultiValue } from "react-select";
-import ReactSelect from "react-select";
+import type { GroupBase, Props, SingleValue, MultiValue, OptionProps } from "react-select";
+import ReactSelect, { components } from "react-select";
```

Comment:
```suggestion
import ReactSelect, { components } from "react-select";
```
The components here - should we be using our internal components [(line 24) ](https://github.com/calcom/cal.com/pull/8623/files#diff-10d1e28a4d05531e2ae96c03a113e6caf2b76270cf5a89fe683e34d2b69512c1R5) instead of the react-select import? I believe this could be the cause of increasing our bundle size pretty drastically